src/common/transformations/src/transformations/common_optimizations/random_uniform_fusion.cpp

```diff
@@ -69,11 +68,8 @@ ov::pass::RandomUniformFusion::RandomUniformFusion() {
         const auto new_ru = ru->clone_with_new_inputs(
             {data, folded_const1 ? folded_const1 : new_mul_add1, folded_const2 ? folded_const2 : new_mul_add2});
 
-        if (pattern_map.count(convert_pattern)) {
-            const auto& convert = pattern_map.at(convert_pattern);
-            const auto cvt = std::dynamic_pointer_cast<ov::op::v0::Convert>(convert.get_node_shared_ptr());
-            if (!cvt)
-                return false;
+        auto cvt = pattern_map.at(random_uniform_or_convert_pattern).get_node_shared_ptr();
```

Comment:
at will throw exception here
please use ".count" method

do not use ".get_node_shared_ptr()" on nullptrs